### ISSUE TITLE
Windows7 screen resolution is not the same as my laptop screen resolution

### ISSUE THREAD

**Issue by 4vrel** [QubesOS/qubes-issues#4764]
### Qubes OS version: R4.0

Hello,
I have installed a windows 7 HVM, witch only give my two choice of screen resolution :  800x600 or 1024x768.
But my screen is 1366x768, and this screen resolution doesn't show up.

### Affected component(s):
Windows 7 HVM

---

### Steps to reproduce the behavior:
Create a HVM template with window 7.

### Expected behavior:
Height and width of the VM feet the onesof my screen.

### Actual behavior:
The VM will have a fixed height and width.
The height might be the same as the one your screen, but the width might be shorter

### General notes:

---

**Comment by andrewdavidwong** [comment]
I'm guessing you've already tried changing the resolution from within Windows' settings inside the HVM?

**Comment by kschubitz** [comment]
Judging from the part where the current screen resolution is just a guess, I assume the answer is "no". Otherwise the resolution should be known.
Just right click the Windows desktop and choose screen resolution, then switch it to whatever suits your needs best.
I suggest to use the qubes-users mailing list for question on how to configure things instead.

**Comment by kschubitz** [comment]
Which version of Windows 7 did you install?

**Comment by 4vrel** [comment]
Yes that's worked ! Thanks !

Actually my window VM was a backup VM that I installed on a new machine, that might be the cause of the bug.

**Comment by BCSharp** [comment]
I had exactly the same problem. While investigating issue #4886, I created a plethora of VM clones and lost track where the video-model was unset. Then was puzzled by the inability to set the screen resolution higher than 1024x768 in some qubes. I've spent some time trying to figure out why, and the official documentation was the first place to consult. 


So I think that a short mention of this effect in the documentation would have greatly helped to connect the dots and am willing to submit a documentation change.
